src/test/scala/cognite/spark/SparkTest.scala

```diff
@@ -51,7 +51,7 @@ trait SparkTest extends CdpConnector {
         actionValue
       },
       Constants.DefaultInitialRetryDelay,
-      Constants.DefaultMaxRetries
+      Constants.DefaultMaxRetries*2
```

Comment:
? :)

Reply:
Seeing quite a lot of retry until failure, so wanted to try out increasing the timeout. Didn't seem to help, I guess the retries until failure were because of errors related to EC giving the wrong results.